custom_model_runner/datarobot_drum/drum/perf_testing.py

```diff
@@ -465,7 +465,7 @@ def _run_test_case(self, tc, results):
         bar.finish()
 
         # TODO: even if prediction request fail we should try and get server stats
-        response = requests.get(self._url_server_address + self._stats_endpoint)
+        response = self.check_server_stats()
 
```

Comment:
Is this a new function from another PR?

Reply:
unintended change, I thought I was going to extract a method here and thought I had reverted everything.